daq/topology.py

```diff
@@ -424,17 +428,18 @@ def _generate_main_acls(self):
         local_acl = []
         acls = {}
 
-        dot1x_pri_ports = []
 
         for devices in self._set_devices.values():
             for device in devices:
                 if device and device.gateway:
-                    dot1x_pri_ports.extend(device.gateway.get_possible_test_ports())
-
-        self._add_acl_rule(incoming_acl, eth_type=self._DOT1X_ETH_TYPE, ports=dot1x_pri_ports)
-
-        self._add_acl_rule(secondary_acl, eth_type=self._DOT1X_ETH_TYPE,
-                           ports=list(range(1, self.sec_port)))
+                    self._add_acl_rule(incoming_acl,
```

Comment:
this nesting is getting ugly -- extract the DO1X specific stuff to a function and call that...

Reply:
Moved to a separate method